config/src/config/consensus_config.rs

```diff
@@ -47,8 +47,8 @@ impl Default for ConsensusConfig {
             safety_rules: SafetyRulesConfig::default(),
             sync_only: false,
             mempool_poll_count: 1,
-            decoupled_execution: false, // by default, we turn of the decoupling execution feature
-            channel_size: 30,           // hard-coded
+            decoupled_execution: true, // by default, we turn of the decoupling execution feature
```

Comment:
remove the switch from the PR

Reply:
I'll do this after we pass the cluster test

Reply:
changed back